locales/ro_RO/LC_MESSAGES/duckduckgo.po

```diff
@@ -3713,16 +3724,20 @@ msgid "Turn off:"
 msgstr "Dezactivează:"
 
 msgid "TV"
-msgstr ""
+msgstr "TV"
 
 msgid "Twitter"
-msgstr ""
+msgstr "Twitter"
 
+# Ambiguous
+#, fuzzy
 msgid "Type %sDuckDuckGo%s in the first %sform field"
-msgstr "Tastează %sDuckDuckGo%s în primul %scâmp al formularului"
+msgstr "Scrie %sDuckDuckGo%s în primul câmp din %s"
 
+# Ambiguous
+#, fuzzy
 msgid "Type %sduckduckgo.com%s in the %ssecond form field"
-msgstr "tip %s rataratadute.com%s in %s al doilea camp din formular"
+msgstr "Scrie %sduckduckgo.com%s în al doilea câmp din %s"
```

Comment:
Can you please show an example for this string?

Reply:
As above :)